drafts/draft-0002.md

```diff
@@ -0,0 +1,118 @@
+1. **Open and Decentralized** - The specification needs to be open in that it can be easily read and implemented by anyone. All the dependencies to implement such a specification should be easily accessible, free of cost, and free from a central authority or service. This will ensure that there is no single point of failure in the system and the system can remain permissionless.
+2. **Standardized** - The specification should not require any special handling for specific operations on the network. This will ensure that there is only a nominal amount of effort needed for developers on both ends (URI generation and signing) to implement all operations on the network. It should also be independent of a specific platform and it should be possible to easily implement in any programming language.
+3. **Secure** - The specification should not expose secret keys in any form. It should include requirements on how wallets can keep their users secure by ensuring that the user has enough visibility into the transaction that she is about to sign.
+4. **Future-Proof** - The specification and its implementation should be designed in a way that allows operations that may be added to the Stellar Network in the future to be easily incorporated into the proposed specification. This will ensure that developers need to make minimal upgrades to their applications and wallets to support additions to the network protocol. The specification should also leave room for expansion to accommodate future use cases that were not anticipated at the time of writing.
+
+## Specification
+A standardized URI Scheme can achieve the goals laid out above. The syntax of the scheme will contain all the information needed to make the necessary payment to the application’s account.
+
+The scheme name will be `stellar` and should always be followed by a colon and two forward slashes to look like this: `stellar://`. The syntax for the URI will look like this: `stellar://<operation>/?<param1>=<value1>&<param2>=<value2>`, where `operation`, and the options for the query params are defined below.
+
+### Operation `tx`
+The `tx` operation represents a request to sign a specific XDR `Transaction`. The parameters for the `tx` op are as follows:
+1. `xdr` (required) -  A Stellar transaction in XDR format that is base64 encoded and then URL-encoded. If the existing source account uses the Master Account behind the network passphrase with an empty string (i.e. `keypair.Master("").Address()`) then the interpreting code will replace the source account and sequence number on this XDR `Transaction` with its own source account and sequence number before it signs it. If the source account is not the sentinel value and the sequence number is 0 then the interpreting code should replace only the sequence number before signing. See the reference implementation for implementation details.
```

Comment:
The sequence number being updated here is that of the source account correct? This should stated clearly.